firmware/quadruna/BMS/src/app/app_heartbeatMonitors.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#include "app_heartbeatMonitor.h"
+
+extern HeartbeatMonitor BMS_heartbeat_monitor;
```

Comment:
I like this - I like this a lot. Nit though: Let's rename this to `app_heartbeatMonitor_cfg.h` or something along that line... While this file is technically a collection of heartbeat monitors, it's more informative to tell the developer that this is where the heartbeat monitor is configured.

Reply:
I think this gets into a question of what to do with all the board specific config files (very much including hardware level stuff). I just named it this way to keep consistent with the way all the other shared library using modules do it.

also i think atm this is just a reference implementation, and we can figure out in the new year what to do with these types of files

Reply:
I'm cool with merging in this state to move things forward then